packages/stark-core/src/modules/user/services/user.service.ts

```diff
@@ -1,3 +1,4 @@
+/* tslint:disable completed-docs*/
```

Comment:
The correct format is: `/* tslint:disable:completed-docs */` as in the file above otherwise all rules are disabled ;)